### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by migo1**
### Milestones covered

- developed a desktop version after mobile-first
- applied media queries

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -114,6 +122,78 @@ header {
...
+  }
+
+  .content {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 12px;
+    width: 39%;
+  }
```
Comment (Shisui6) [inline style.css]: Kindly increase the width of the text in the headline section to match the figma design better🙏

Your design:
image

Figma design:
image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @migo1, 

# Requires Changes 🍀
You have done an amazing job!👏👏
Your design is really good 🖼️, and your code follows best practices. There's just one small problem we need to fix🪲.

## Highlights
- Great use of media queries ✅
- Beautiful design ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (migo1) [reply style.css]: @Shisui6 noted, working on it

**Review by TadesseDev — APPROVED** [review]
### Hi @migo1 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by migo1** [comment]
@TadesseDev, that's literally me on GIF right now, thanks for the code review and approval .

**Comment by TadesseDev** [comment]
Keep up the good work @migo1

**Pull request merged**